rdrf/explorer/migrations/0012_drop_explorer_models.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.15 on 2022-12-05 13:27
+
```

Comment:
Does this happen in two stages? Merge in this migration, then delete it? Or just keep around an empty stub of a django app?

Reply:
I've got a part one and part two on the JIRA, because django won't let me run the final 'drop' migration without keeping the explorer app.

https://eresearchqut.atlassian.net/browse/ERE-1848